fml_manager/fml_manager/utils/fate_builders.py

```diff
@@ -462,3 +462,429 @@ def with_components(self, *components):
+        :return:
+        """
+        self.host_args_data_train_data.append(train_data)
+        return self
+
+    def with_host_dataio_0_with_label(self, label: bool) -> object:
+        """
+
+        :param label:
+        :return:
+        """
+        self.host_dataio_0_with_label.append(label)
+        return self
+
+    def with_host_dataio_0_output_format(self, format: str) -> object:
+        """
+
+        :param format:
+        :return:
+        """
+        self.host_dataio_0_output_format.append(format)
+        return self
+
+    def build(self) -> dict:
+        """
+
+        :return:
+        """
+        body = dict()
+        guest = dict()
+        host = dict()
+        if self.guest_args_data_train_data is None:
+            pass
+        else:
+            temp = dict(
+                args=dict(
+                    data=dict(
+                        train_data=self.guest_args_data_train_data
+                    )
+                )
+            )
+            deepSearch(guest, temp)
+        if len(self.guest_dataio_0_with_label) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    with_label=self.guest_dataio_0_with_label
+                )
+            )
+            deepSearch(guest, temp)
+
+        if len(self.guest_dataio_0_label_name) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    label_name=self.guest_dataio_0_label_name
+                )
+            )
+            deepSearch(guest, temp)
+
+        if len(self.guest_dataio_0_label_type) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    label_type=self.guest_dataio_0_label_type
+                )
+            )
+            deepSearch(guest, temp)
+
+        if len(self.guest_dataio_0_output_format) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    output_format=self.guest_dataio_0_output_format
+                )
+            )
+            deepSearch(guest, temp)
+
+        if self.host_args_data_train_data is None:
+            pass
+        else:
+            temp = dict(
+                args=dict(
+                    data=dict(
+                        train_data=self.host_args_data_train_data
+                    )
+                )
+            )
+            deepSearch(host, temp)
+        if len(self.host_dataio_0_with_label) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    with_label=self.host_dataio_0_with_label
+                )
+            )
+            deepSearch(host, temp)
+
+        if len(self.host_dataio_0_output_format) != 0:
+            temp = dict(
+                dataio_0=dict(
+                    output_format=self.host_dataio_0_output_format
+                )
+            )
+            deepSearch(host, temp)
+
+        body["host"] = host
+        body["guest"] = guest
+        return body
+
+
+class AlgorithmParametersBuilder(object):
```

Comment:
should add the [reference](https://github.com/FederatedAI/FATE/blob/master/federatedml/conf/default_runtime_conf/logistic_regression_param.json) for usage